recursion.rb

```diff
@@ -1,14 +1,42 @@
 
 def fact(n)
+  return 1 if n <= 1
+  n*fact(n-1)
 end
```

Comment:
I like your choice of "n <= 1" instead of "n==1" here - just in case someone calls it with zero or a negative number, your code won't recurse forever. Good solution.